zenoh/src/net/runtime/orchestrator.rs

```diff
@@ -911,7 +911,7 @@ impl Runtime {
         }
     }
 
-    /// Returns `true` if a new Transport instance has been opened with `zid`.
+    /// Returns `true` if a new Transport instance exists between us and `zid`.
```

Comment:
This is misleading, as the `connect` function doesn't only check if a transport exists, but also tries to open one if it does not exist.

Reply:
```suggestion
    /// Opens a new transport with `zid`.
    /// Returns `true` if successful, or if a Transport instance already exists with `zid`.
```